Hippo.Core/Domain/Cluster.cs

```diff
@@ -20,12 +20,20 @@ public class Cluster
         [Required]
         [MaxLength(250)]
         public string Description { get; set; } = String.Empty;
+        [MaxLength(250)]
+        public string SshName { get; set; } = String.Empty;
+        [MaxLength(40)]
+        public string SshKeyId { get; set; } = String.Empty;
+        [MaxLength(250)]
+        public string SshUrl { get; set; } = String.Empty;
+        public bool IsInactive { get; set; }

```

Comment:
Default to true?

Reply:
Oh, you did IsInactive.
We usually do IsActive...

Reply:
Why would you create a cluster unless you intend it to be active?

Reply:
Also, if you create an inactive cluster, it will be automatically filtered out of results.

Reply:
I didn't read your var name correctly. All our other apps use IsActive name